ccd-definition/CaseEventToComplexTypes/upload-CMO/uploadCMO.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "LiveFrom": "01/01/2017",
-    "ID": "SupportingEvidenceBundle",
+    "ID": "UploadedCaseManagementOrder",
```

Comment:
Don't think these first three needed to change the id, the `cmoSupportingDocs` is a separate case field which is a collection of `SupportingEvidenceBundle`